src/main/views/template.njk

```diff
@@ -85,7 +85,7 @@
           text: language
       },
       {
-          href: "/Cookie-policy",
```

Comment:
Can we keep cookie policy name or it was requirement to have it cookie?

Reply:
I just used the same naming scheme as the gov notify cookies page, can switch it back to cookie policy, no worries :)

Reply:
Still need to revert this

Reply:
Cheers, well spotted!